src/components/character-count/index.md.njk

```diff
@@ -97,6 +97,16 @@ For example, ‘Enter a summary’.
 Say ‘[whatever it is] must be [number] characters or fewer’.<br>
 For example, ‘Summary must be 400 characters or fewer’.
 
+### Consider the effect of emojis and accented characters
+
+Using emojis and accented characters can affect character counts in a way you might not want.
+
+The character count component uses the `string.length` property to work out how long the user input is.
+
+If the user includes an emoji or an accented character, the component will count its code units, not its characters.
+
+For example, if the user typed 'cat' and then added a cat emoji, the count would be 5. This is because the component counts the emoji as 2 code units.
```

Comment:
Is there anything we should advise users to do? Like telling _their_ users about emojis and accented characters inflating the count?